shared/fs/row/common.js

```diff
@@ -73,8 +73,8 @@ const pathItemActionIcon = {
 
 const badgeContainer = {
   position: 'absolute',
-  left: Styles.isMobile ? -24 : 24,
-  top: Styles.isMobile ? -20 : -1,
+  left: Styles.isMobile ? -28 : 24,
+  top: Styles.isMobile ? -4 : -1,
```

Comment:
did you test this on both iOS and Android?

Reply:
I tested on iOS. You mean Android would have different layout?

Reply:
I'm not sure. I originally came up with these offsets via trial and error on the iOS simulator, so I'm not sure how they could be wrong.

Reply:
Perhaps they were just affected by some other changes. Anyway I'm building android now and will try those stories on Android